tests/src/test/resources/definition.yaml

```diff
@@ -5,18 +5,18 @@ apis:
     production_endpoint:
       type: 'load_balance'
       urls:
-        - 'https://petstore.ballerina.io/'
-        - 'https://petstore.ballerina.io/'
+        - 'http://www.mocky.io/v2/5cbfdeb03100006600035ef1'
```

Comment:
This will make the internet connection required for running tests I guess?

Reply:
fixed it